hydro/distribution.yaml

```diff
@@ -3883,6 +3883,17 @@ repositories:
       type: git
       url: https://github.com/UC3MSocialRobots/maggie_devices_msgs.git
       version: hydro-devel
+    release:
+      packages:
+      - eyelids_msgs
+      - ir_controller_msgs
+      - maggie_devices_msgs
+      - motor_controller_msgs
+      - rfid_msgs
```

Comment:
Now I think we are following the Package Naming Conventions for our packages. Please tell me if everything is ok or not. Thanks.